fdm-app/.env.example

```
@@ -135,4 +135,16 @@ PUBLIC_POSTHOG_HOST= # Example: https://eu.i.posthog.com
+POSTMARK_SENDER_NAME=
+
+# -------------------------------------
+# LangSmith Tracing (optional, development)
+# -------------------------------------
+# Enable LangSmith tracing for full observability of agent reasoning,
+# tool calls, token usage, and latency. No code changes required —
+# set these env vars and all LangGraph runs are automatically traced.
+# Obtain an API key at https://smith.langchain.com/
+# Required: No (omit or leave empty to disable)
+LANGSMITH_TRACING=true
+LANGSMITH_API_KEY=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The LANGSMITH_API_KEY key should go before the LANGSMITH_TRACING key

(UnorderedKey)